prelude/__internal__/Show.mad

```diff
@@ -1,9 +1,6 @@
 /**
+ * Convert an instance to a string 
```

Comment:
I can pitch that at some point but I think the term instance is wrong here. A more correct way of putting it would be: It converts a value of a type that implements Show to a String.

Reply:
What about: "An abstract (to-be-defined) implementation of converting a type instance to a String"?

Reply:
Abstract could work as that is definitely not concrete. But if we have an introduction to interfaces I think we can omit that part. I don’t think we have to document language features in the API docs. So I’d just describe what the methods it defines do, what do you think?

Reply:
Yeah, I think it's important to document but maybe not in the API docs.